.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+- If applicable describe the current behaviour and explain which behaviour you expected to see instead.
+
+**Describe the solution you'd like and why**
```

Comment:
White spacing or maybe a way to segregate title from content beyond bold.

Reply:
done